backend/sns/users/controller.py

```diff
@@ -415,6 +415,31 @@ def read_followings(
     )
 
 
+@router.get(
+    "/users/{user_id}/follow",
+    response_model=bool,
+    status_code=status.HTTP_200_OK,
+)
+def read_is_followed(
+    user_id: int,
+    user_service: UserService = Depends(UserService),
```

Comment:
type annotaion 클래스와 Depends 안에 들어가는 값이 같을 때에는 생략할 수 있습니다:

```suggestion
    user_service: UserService = Depends(),
```

물론, 생략하지 않아도 됩니다.